oak_functions_freestanding_bin/build.rs

```diff
@@ -15,7 +15,6 @@
 //
 
 fn main() {
-    println!("cargo:rerun-if-changed=target.json");
-    println!("cargo:rerun-if-changed=layout.ld");
-    println!("cargo:rustc-link-arg=--script=layout.ld");
+    println!("cargo:rerun-if-changed=../oak_restricted_kernel/layout.ld");
+    println!("cargo:rustc-link-arg=--script=../oak_restricted_kernel/layout.ld");
```

Comment:
Could these be added to the kernel crate directly, and use something like the following to locate those files? Or even better, can it just be part of the build script of that crate, so that other crates don't even have to have build scripts themselves? Or does rustc-link-arg only apply to the crate that declares it?

https://github.com/project-oak/oak/blob/8783bce33212f755e70b39fc1513cb79b68c4377/oak_protobuf_idl_tests/build.rs#L19-L22

Reply:
Moving the `build.rs` under the kernel doesn't work.

However, using `WORKSPACE_ROOT` works. I was rather surprised at that, as the freestanding binaries are not part of our global workspace. I guess that environment variable comes from elsewhere.